tests/phpunit/Unit/functions/StrToHumanReadableTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Cig\Tests\Unit\Functions;
+
+class StrToHumanReadableTest extends \Cig\Tests\Unit\BaseTestCase {
+
+	/***
+	 * @return array Strings, expected results
+	 */
+	public function provide_str_case_data(): array {
+		return [
+			'camelCase' => [
+				// string
+				'camelCaseStringTest',
+				//  expected result
+				'camel Case String Test'
+			],
+			// note: not technically a 'fail' but incorrectly separates surnames with bicapital
+			// TODO: add option to pass in lowercase var for str_to_words?
+			'bicapital' => [
+				// string
+				'danny-DeVito_',
+				//  expected result
+				'danny De Vito'
+			],
+			'pascal' => [
+				// string
+				'PascalCaseIsTheCase',
+				//  expected result
+				'Pascal Case Is The Case'
+			],
+			'snake' => [
+				// string
+				'itsa_me_snake_case',
+				//  expected result
+				'itsa me snake case'
+			],
+			'kebab' => [
+				// string
+				'kebab-case-pepper-tomato',
+				//  expected result
+				'kebab case pepper tomato'
+			],
+			//TODO: another instance of integers turning into strings: expected or should this fail?
+			'integer' => [
```

Comment:
I'd make this it's own test like you did for `strToCamelCaseTest.php` and add `declare(strict_types=1);` at the top.

The reason I'd move it out is because we do want it to fail when given an integer and the test isn't setup (nor should it be) to handle both successful and failed calls.